ontology/spdx-ontology.owl.xml

```diff
@@ -9,14 +9,15 @@
      xmlns:doap="http://usefulinc.com/ns/doap#"
      xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
     <owl:Ontology rdf:about="http://spdx.org/rdf/terms">
-        <owl:versionIRI rdf:resource="http://spdx.org/rdf/terms/2.2"/>
-        <rdfs:comment>This specification describes the SPDX® language, defined as a dictionary of named properties and classes using W3C&apos;s RDF Technology.
-
-SPDX® is a designed to allow the exchange of data about software packages. This information includes general information about the package, licensing information about the package as a whole, a manifest of files contained in the package and licensing information related to the contained files.
-
-Known issues:</rdfs:comment>
-        <rdfs:label>SPDX 2.2</rdfs:label>
-        <owl:versionInfo rdf:datatype="http://www.w3.org/2001/XMLSchema#decimal">2.2</owl:versionInfo>
+        <owl:versionIRI rdf:resource="http://spdx.org/rdf/terms/2.3"/>
+        <rdfs:comment xml:lang="en">This specification describes the SPDX® language, defined as a dictionary of named properties and classes using W3C&apos;s RDF Technology.
+
+SPDX® is an open standard for communicating software bill of material information, including components, licenses, copyrights, and security references. SPDX reduces redundant work by providing a common format for companies and communities to share important data, thereby streamlining and improving compliance.
+.
+Known issues:
+- rdfs:comment and rdfs:seeAlso are used within the SPDX classes and causes a redefinition of these standard terms</rdfs:comment>
+        <rdfs:label>SPDX 2.3</rdfs:label>
+        <owl:versionInfo rdf:datatype="http://www.w3.org/2001/XMLSchema#decimal">2.3</owl:versionInfo>
```

Comment:
Why is the version a decimal? I'd suggest the datatype is deleted, by default it's a string.

Reply:
@zvr I agree - however, whenever I try to change this in Protege tool, it keeps changing it back to a decimal type.  I looked up the W3C OWL spec and there is no requirement for this to be a decimal datatype.

Let me know if you find a way to fix this (other than manually patching it after editing in Protege).